src/app/utils/tests/assert.spec.ts

```diff
@@ -9,4 +9,16 @@ describe('assert', () => {
         expect(() => assert(false, 'error')).toThrowError('Assertion failure: error');
         expect(ErrorLoggerService.logError).toHaveBeenCalledWith('Assertion failure', 'error');
     });
+    it('should console.log the second parameter when provided', () => {
+        // Given the function
```

Comment:
euuuh c'pas un bon given ça ? ou alors mets clairement "the assert function", mais "the function" c'est vague et un peu tautologique ?